pkg/controller/neutronovsagent/neutronovsagent_controller.go

```diff
@@ -345,20 +373,12 @@ func newDaemonset(cr *neutronv1.NeutronOvsAgent, cmName string, configHash strin
                                 Name:  "CONFIG_HASH",
                                 Value: configHash,
                         },
-                },
-                VolumeMounts: []corev1.VolumeMount{
-                        {
-                                Name:      cmName,
-                                ReadOnly:  true,
-                                MountPath: "/etc/neutron/neutron.conf",
-                                SubPath:   "neutron.conf",
-                        },
                         {
-                                Name:      cmName,
-                                ReadOnly:  true,
-                                MountPath: "/etc/neutron/plugins/ml2/openvswitch_agent.ini",
-                                SubPath:   "openvswitch_agent.ini",
+                                Name:  "INIT_CONFIG_HASH",
```

Comment:
Shouldn't we add the initConfigHash to the ENV parameters above?

Reply:
it is in the ENV section. the volumemounts begins now at 381.

Reply:
sounds good. lgtm